docs/pages/get-started/get-started.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: "Get Started with Teleport"
+description: "Learn how to deploy a cluster, connect infrastructure, set up access controls, and review audit logs."
+tocDepth: 3
+template: no-toc
+tags:
+  - get-started
+---
+
+import Button from '@site/src/components/Button';
+import Icon from '@site/src/components/Icon';
+import TeleportEditionsGrid from '@site/src/components/Pages/GetStarted/TeleportEditionsGrid';
+import TeleportEditionCard from '@site/src/components/Pages/GetStarted/TeleportEditionCard';
+import FeatureMatrixBanner from '@site/src/components/Pages/GetStarted/FeatureMatrixBanner';
+
+Teleport is an identity-based access platform that secures servers, Kubernetes clusters, databases, internal applications, and desktops using short-lived certificates, detailed audit logging, and fine-grained role-based access controls tied to your SSO provider (e.g., Okta, GitHub, Google Workspace).
+
+This guide walks you through four essential steps to get you up and running.
+
+## Choose your Teleport edition
+
+There are several editions of Teleport that differ in their feature sets, deployment options, and support.
+
+Before choosing one, ask yourself:
```

Comment:
We don't have to block on this, but I still think we can simplify the UX here. Currently, the user needs to:
- Read a bullet
- Figure out that they need to read the information boxes
- Click an information box that corresponds to a bullet

Would it make sense to include the link and description for each edition within the same component so the reader doesn't need to connect so many dots? We could remove the edition boxes and add a short (< 1 sentence) description of each edition to the bullets that precede them, along with a link to each edition page.